src/app/hooks/AuthProvider.tsx

```diff
@@ -45,6 +45,7 @@ type AuthContextValue = {
 	logOut: () => void;
 	permissions: string[];
 	TokenUtils: ReturnType<typeof createEgoUtils>;
+	userPrograms: string[];
```

Comment:
Added in early stages of PR to filter by logged in user permissions, may no longer be necessary

Reply:
Removed